### ISSUE TITLE
[Error]: Module not found

### ISSUE THREAD

**Issue by NitinCopods** [cyntler/react-doc-viewer#274]
I am getting following module not found error. I am using Next JS 14.
![Uploading Screenshot 2024-06-13 at 4.35.06 PM.png…]()

**Comment by cyntler** [comment]
@NitinCopods Is it with the latest version?

**Comment by karimdaghari** [comment]
Indeed. Both `Next` (using turbo) and `react-doc-viewer` are on latest versions.

**Comment by NitinCopods** [comment]
> @NitinCopods Is it with the latest version?

@cyntler Yes, It doesn't work with turbo. So I have removed turbo and currently running my app using webpack and it's working.

**Comment by karimdaghari** [comment]
> @cyntler Yes, It doesn't work with turbo. So I have removed turbo and currently running my app using webpack and it's working.

`v1.14.1` works with turbo tho.

**Comment by NitinCopods** [comment]
> > @cyntler Yes, It doesn't work with turbo. So I have removed turbo and currently running my app using webpack and it's working.
> 
> `v1.14.1` works with turbo tho.

@karimdaghari Okay, I'll create a sandbox for the same and attach it.

**Comment by cyntler** [comment]
OK, so I have to fix module building system again...

ESM, CJS + Webpack, Vite, Turbo, Next, CRA.
OMG! Each tool processes it all differently, I think.

Module support in the JS ecosystem is a bit lacking in my opinion...

**Comment by cyntler** [comment]
@NitinCopods Could you check this problem with v1.16.5?

**Comment by karimdaghari** [comment]
> @NitinCopods Could you check this problem with v1.16.5?

Although not the pinged perso, I'm also concerned. And, unfortunately the (same, exact) problem still persists. Using the same setup Next `14.2.4` and `1.16.5`

**Comment by cyntler** [comment]
> > @NitinCopods Could you check this problem with v1.16.5?
> 
> Although not the pinged perso, I'm also concerned. And, unfortunately the (same, exact) problem still persists. Using the same setup Next `14.2.4` and `1.16.5`

Thanks, ok! I have to prepare this use-case and check it again

**Comment by regexyl** [comment]
A related thread: https://github.com/wojtekmaj/react-pdf/issues/1822
